### PR TITLE
docs/helm: update documentation according reviews

### PR DESCRIPTION
**Pull request by zonggen**
In response to
https://github.com/openshift/console/pull/10467/files#diff-9792ce17aec3657e6f182e08e62ce981cfbcfc33f70cfde0b6f2c40beec672d7,
updating and refining documentation for helm API.

Signed-off-by: Allen Bai <abai@redhat.com>

### CODE REVIEW

**Comment by zonggen** [comment]
@Srivaralakshmi PTAL and let me know if there's anythign I am missing.

File: docs/helm/configure-namespaced-helm-repos.md [hunk 1]
```diff
@@ -1,85 +1,55 @@
...
   # required: chart repository URL
   connectionConfig:
     url: HELM_CHART_REPO_URL
 ```
 
-## Getting Started with `ProjectHelmChartRepository`
+## Adding Namespace-scoped Custom Helm Chart Repositories
 
-To add a new namespace-scoped Helm repository `stable` to a desired namespace `my-namespace`:
+To add a new namespace-scoped Helm repository, add a custom Helm Chart Repository CR, for example, `azure-sample-repo` to your `my-namespace` namespace:
```
Comment (Srivaralakshmi) [inline docs/helm/configure-namespaced-helm-repos.md:32]:
```suggestion
To add a new namespace-scoped Helm repository, add a custom Helm Chart Repository CR, for example, `azure-sample-repo` CR to your `my-namespace` namespace:
```

File: docs/helm/configure-namespaced-helm-repos.md [hunk 2]
```diff
@@ -1,85 +1,55 @@
...
 
 ```bash
 $ cat <<EOF | oc apply --namespace my-namespace -f -
```
Comment (Srivaralakshmi) [inline docs/helm/configure-namespaced-helm-repos.md:35]: Suggestion: I believe that in upstream we use only `kubectl` and not `oc`. `oc` is only for downstream. (Global change for all such `oc` occurrences)

File: docs/helm/configure-namespaced-helm-repos.md [hunk 3]
```diff
@@ -1,85 +1,55 @@
...
-$ kubectl get helmchartrepositories
-NAME          AGE
-cluster       3h30m
-custom-cluster        1m
-
-$ kubectl get projecthelmchartrepositories --namespace my-namespace
-NAME          AGE
-custom-namespace     1m
-```
+The addition of namespace-scoped Helm repository does not impact behavior of the existing cluster-scoped Helm repository.
```
Comment (Srivaralakshmi) [inline docs/helm/configure-namespaced-helm-repos.md:51]:
```suggestion
The addition of the namespace-scoped Helm repository does not impact the behavior of the existing cluster-scoped Helm repository.
```

File: docs/helm/configure-namespaced-helm-repos.md [hunk 4]
```diff
@@ -1,85 +1,55 @@
...
 
 ## API Endpoint `/api/helm/charts/index.yaml`
 
-The `/api/helm/charts/index.yaml` endpoint supports a query parameter `namespace`. For example, a GET request to /api/helm/charts/index.yaml?namespace=my-namespace will respond an aggregated index.yaml file with entities extracted from both cluster scoped Helm repository and Helm repositories in `my-namespace` namespace.
+The `/api/helm/charts/index.yaml` endpoint supports a query parameter `namespace`. For example, a GET request to /api/helm/charts/index.yaml?namespace=my-namespace will respond an aggregated index.yaml file with entities extracted from both cluster-scoped and namespace-scoped Helm repositories.
```
Comment (Srivaralakshmi) [inline docs/helm/configure-namespaced-helm-repos.md:55]:
```suggestion
The `/api/helm/charts/index.yaml` endpoint supports a `namespace` query parameter. For example, a GET request to `/api/helm/charts/index.yaml?namespace=my-namespace` will respond an aggregated `index.yaml` file with entities extracted from both cluster-scoped and namespace-scoped Helm repositories.
```

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
Hi @zonggen: Thanks for implementing my review comments. The content looks good. However, I have left a few minor suggestions. PTAL. Thanks again :)

File: docs/helm/configure-namespaced-helm-repos.md [hunk 5]
```diff
@@ -1,85 +1,55 @@
-# Configure Namespace Scoped Chart Repositories using namespaced CRDs
+# Configure Namespace-scoped Helm Chart Repositories Using Namespaced CRDs
 
-While the cluster-scoped CRD `HelmChartRepository` for Helm repository provides ability for admins to add Helm repositories as custom resources, regular users/developers need an admin's permission in order to add Helm repositories of their choice.
+While the cluster-scoped [HelmChartRepository](https://github.com/openshift/api/blob/master/helm/v1beta1/0000_10-helm-chart-repository.crd.yaml) CRD for Helm repository provides the ability for admins to add Helm repositories as custom resources, regular users/developers need an admin's permission in order to add Helm repositories of their choice.
 
-To avoid permission elevation, use namespace scoped CRD `ProjectHelmChartRepository`.
+The namespace-scoped `ProjectHelmChartRepository` custom resource allows project members with the appropriate RBAC permissions to create Helm repository resources of their choice but scoped to their namespace. Such project members can see charts from both clusters-scoped and namespace-scoped Helm repositories.
```
[the remark this answers is not in the document]
Comment (zonggen) [reply docs/helm/configure-namespaced-helm-repos.md]: Updated :+1:

File: docs/helm/configure-namespaced-helm-repos.md (hunk 1 above)
Comment (zonggen) [reply docs/helm/configure-namespaced-helm-repos.md]: Updated :+1:

File: docs/helm/configure-namespaced-helm-repos.md (hunk 2 above)
Comment (zonggen) [reply docs/helm/configure-namespaced-helm-repos.md]: Updated :+1:

File: docs/helm/configure-namespaced-helm-repos.md (hunk 3 above)
Comment (zonggen) [reply docs/helm/configure-namespaced-helm-repos.md]: Updated :+1:

File: docs/helm/configure-namespaced-helm-repos.md (hunk 4 above)
Comment (zonggen) [reply docs/helm/configure-namespaced-helm-repos.md]: Updated :+1:

**Comment by zonggen** [comment]
@Srivaralakshmi Thanks for the reviews! I've updated according to your comments and please let me know if more changes are needed.

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@zonggen I have left some minor suggestions. PTAL. Thanks for addressing all the review comments.

**Comment by zonggen** [comment]
@Srivaralakshmi Thanks for the look, I've updated according to your comments.

**Review by Srivaralakshmi — APPROVED** [review]
Thanks, @zonggen LGTM :)

**Pull request merged**